src/App.tsx

```diff
@@ -17,18 +74,23 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                setQuery={onSetQuery}
+                filterBy={filterBy}
+                setFilterBy={onSetFilterBy}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {!todos.length
+                ? <Loader />
+                : <TodoList todos={visibleTodos} viewModule={viewModule} />}
             </div>
           </div>
         </div>
       </div>
-
-      <TodoModal />
+      {selectTodoId && (<TodoModal selectTodo={selectTodo} viewModule={viewModule} />)}
```

Comment:
```suggestion
      {!!selectTodoId && (<TodoModal selectTodo={selectTodo} viewModule={viewModule} />)}
```

<img width="364" alt="Screenshot 2022-10-04 at 20 31 39" src="https://user-images.githubusercontent.com/47606379/193887143-54d053ba-4f29-4269-943d-caaea467f1ab.png">